third_party/docfx/templates/devsite/partials/class.header.tmpl.partial

```diff
@@ -48,8 +48,10 @@
 {{#derivedClasses.0}}
 </div>
 {{/derivedClasses.0}}
+{{#assemblies.0}}
 <h6><strong>{{__global.namespace}}</strong>: {{{namespace.specName.0.value}}}</h6>
 <h6><strong>{{__global.assembly}}</strong>: {{assemblies.0}}.dll</h6>
+{{/assemblies.0}}

```

Comment:
It looked good on the goldens so I may have prematurely approved, but is it safe to assume that namespace exists if there's an assembly for it? Or can we have a case where there's a namespace but no assembly?

Reply:
Goldens are working, but you're right I'm not sure we can make that assumption. I'll update so we check for namespace.specName.0.value to show first part and check for assemblies to show the second part.